css/portal-dashboard/student-names.less

```diff
@@ -4,7 +4,7 @@
   display: inline-block;
   width: 240px;
 
-  .studentName {
+  .studentNameRow {
```

Comment:
This name change breaks some of the styling where the expectation is that the class is naked `studentName` - compact mode, for example is now broken.